### PR TITLE
show basic article list on main page only

### PR DESCRIPTION
**Pull request by Isaac-Sturtridge**
Complete 4 CORE: View a list of all articles

Only basic styling and layout.

As this is the first route, no routes yet, this is just the main page.

### CODE REVIEW

File: src/components/ArticleList.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            <article className="article_card">
+        <h2>{article.title}</h2>
+        <img src={article.article_img_url} alt={article.title} />
+        <h3>{article.author}</h3>
+        <h4>{article.topic}</h4>
+        <p>Votes: {article.votes} Comment Count: {article.comment_count} Created at: {article.created_at}</p>
+        </article>
+        )
+      })}
+    </section>
```
Comment (nc-linda-guester) [inline src/components/ArticleList.jsx]: So this does absolutely work, the task does however specify we are looking to see an _articleCard_ - due to that, I would expect this logic to essentially be its own articleCard component!

File: src/components/ArticleList.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://iz-nc-news.onrender.com/api/articles")
```
Comment (nc-linda-guester) [inline src/components/ArticleList.jsx]: For the sake of readability, I would definitely suggest to pull out anything to do with axios into a separate utils folder/file - then you can utilize an axios.create and separate out the concerns into separate functions you can then import!

**Review by nc-linda-guester — CHANGES_REQUESTED** [review]
A few things to change to be more in line with the task at hand, as well as a small change that will really help with readability later - see comments :)

File: src/utils/axios.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://iz-nc-news.onrender.com/api",
+});
+
+const getArticles = api.get("/articles");
+
```
Comment (Harrison365) [inline src/utils/axios.js:8]: Nice 👍

File: src/components/ArticleList.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles
+      .then((response) => {
+        return response.data;
+      })
+      .then((data) => {
+        setArticles(data.articles);
+      });
```
Comment (Harrison365) [inline src/components/ArticleList.jsx:17]: Perhaps add some isLoading functionality here :)

**Pull request merged**